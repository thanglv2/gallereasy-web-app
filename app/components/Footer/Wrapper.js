
import styled from 'styled-components';

const Wrapper = styled.footer`
  display: flex;
  justify-content: space-between;
  align-items: center;
  position: absolute;
  height: 50px;
  bottom: 0;
  width: 100%;
  background-color: ${props => props.theme.darkgrey};
  span {
    color: ${props => props.theme.dimgrey};
    font-weight: 500;
    margin: 0 50px;
  }
  @media(max-width: 600px) {
    span {
      margin: 0 10px;
    }
  }
`;

export default Wrapper;