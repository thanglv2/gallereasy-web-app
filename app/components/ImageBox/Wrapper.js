import styled from 'styled-components';

const Wrapper = styled.div`
    margin: 0 auto;
    position: relative;
    div {
      visibility: ${props => props.isFavourite ? 'visible' : 'hidden'};
    }
    &:hover {
      div {
        visibility: visible;
        opacity: ${props => props.isFavourite ? 1 : 0.4};
      }
      cursor: pointer;
    }
`;

export default Wrapper;
