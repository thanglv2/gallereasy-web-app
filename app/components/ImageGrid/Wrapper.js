import styled from 'styled-components';

const Wrapper = styled.div`
    display: grid;
    margin: 50px auto;
    max-width: 1000px;
    grid-template-columns: repeat(4, minmax(200px, 1fr));
    grid-gap: 20px;
    align-items: center;
    img {
      border: 1px solid #ccc;
      box-shadow: 2px 2px 6px 0px rgba(0,0,0,0.3);
      object-fit: cover;
      width: 200px;
      height: 180px;
    }
`;

export default Wrapper;