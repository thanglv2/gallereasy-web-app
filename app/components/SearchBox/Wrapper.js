
import styled from 'styled-components';

const Wrapper = styled.div`
  text-align: center;
  input {
    border-top-style: hidden;
    border-right-style: hidden;
    border-left-style: hidden;
    border-bottom-style: groove;
    outline: none;
    font-size: 30px;
    background-color: ${props => props.theme.bg}
    padding-bottom: 10px;
    &::placeholder {
      color: ${props => props.theme.grey};
      font-weight: normal;
    }
  }
`;

export default Wrapper;