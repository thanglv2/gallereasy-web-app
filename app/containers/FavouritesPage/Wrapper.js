import styled from 'styled-components';

const Wrapper = styled.div`
  width: 100%;
  margin: 50px auto 50px;
  padding-bottom: 10px;
`;

export default Wrapper;