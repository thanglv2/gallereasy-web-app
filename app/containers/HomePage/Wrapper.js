import styled from 'styled-components';

const Wrapper = styled.div`
  width: 100%;
  margin: 50px auto 100px;
`;

export default Wrapper;