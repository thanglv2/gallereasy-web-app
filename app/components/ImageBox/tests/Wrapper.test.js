import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import Wrapper from '../Wrapper';

describe('<Wrapper />', () => {
  it('render without error', () => {
    const tree = renderer.create(<Wrapper isFavourite />).toJSON()
    expect(tree).toMatchSnapshot()
  });

  it('render without error', () => {
    const tree = renderer.create(<Wrapper isFavourite={false} />).toJSON()
    expect(tree).toMatchSnapshot()
  });
});