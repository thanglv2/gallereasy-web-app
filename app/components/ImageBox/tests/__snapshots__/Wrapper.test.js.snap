// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper /> render without error 1`] = `
.c0 {
  margin: 0 auto;
  position: relative;
}

.c0 div {
  visibility: visible;
}

.c0:hover {
  cursor: pointer;
}

.c0:hover div {
  visibility: visible;
  opacity: 1;
}

<div
  className="c0"
/>
`;

exports[`<Wrapper /> render without error 2`] = `
.c0 {
  margin: 0 auto;
  position: relative;
}

.c0 div {
  visibility: hidden;
}

.c0:hover {
  cursor: pointer;
}

.c0:hover div {
  visibility: visible;
  opacity: 0.4;
}

<div
  className="c0"
/>
`;
